packages/expo-dev-launcher/ios/EXRCTAppDelegateInterceptor.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+#import <EXDevMenu/DevClientAppDelegate.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface EXRCTAppDelegateInterceptor : DevClientAppDelegate
+
+@property(nonatomic, weak) id<RCTBridgeDelegate> bridgeDelegate;
+@property(nonatomic, weak) id<RCTBridgeDelegate> interceptor;
+
+- (instancetype)initWithBridgeDelegate:(id<RCTBridgeDelegate>)bridgeDelegate interceptor:(id<RCTBridgeDelegate>)interceptor;
+
+@end
+
+NS_ASSUME_NONNULL_END
```

Comment:
what's the difference of this and the `packages/expo-modules-core/ios/ReactDelegates/EXRCTBridgeDelegateInterceptor.h`? could we just keep one?

Reply:
The only difference is that this inherits from `DevClientAppDelegate`, that way we can call `createBridgeAndSetAdapterWithLaunchOptions` inside `ExpoDevLauncherReactDelegateHandler`